requirements-rtd.txt

```diff
@@ -5,5 +5,5 @@ readthedocs-sphinx-search==0.1.2
 sphinx-toggleprompt==0.2.0
 sphinx-copybutton==0.5.0
 roiextractors  # Needed for the API documentation
-spikeinterface  # Needed for the API documentation
+spikeinterface==0.101.0  # Needed for the API documentation
```

Comment:
Why pinning instead of lower `>=` ?

Reply:
I followed the other pins, used >= for the main requirements